src/pages/editor/ChannelClaimTable.tsx

```diff
@@ -0,0 +1,42 @@
+import { Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { ClaimedChannel, SelectedPublicationChannelType } from '../../types/customerInstitution.types';
+import { PublicationChannelType } from '../../types/registration.types';
+import { ChannelClaimTableRow } from './PublicationChannelTableRow';
+
+interface ChannelClaimTableProps {
+  channelClaimList: ClaimedChannel[];
+  channelType: SelectedPublicationChannelType;
```

Comment:
Siden `SelectedPublicationChannelType` bare brukes her burde den kanskje defineres her også? Ser ut som det er en verdi som brukes for å definere hvilken type denne lista skal vise og ikke noe annet.
Får behov for lignende i min løsning hvor man skal claime serie/tiddsskrift også, og da har jeg tenkt på en løsning ala dette:

```suggestion
  channelType: Publisher["type"] | SerialPublication["type"];
```

Da vet vi at typen her samsvarer med det som faktisk er i datamodellen også